modules/govuk/manifests/apps/service_manual_frontend.pp

```diff
@@ -0,0 +1,32 @@
+# == Class: govuk::apps::service_manual_frontend
+#
+# Service Manual Frontend is an app to serve content pages for the service manual from the content store
+#
+# === Parameters
+#
+# [*vhost*]
+#   Virtual host used by the application.
+#   Default: 'service-manual-frontend'
+#
+# [*port*]
+#   What port should the app run on?
+#
+class govuk::apps::service_manual_frontend(
+  $vhost = 'service-manual-frontend',
+  $port = '3090',
```

Comment:
Add a line in [this file](https://github.gds/gds/development/blob/master/Procfile)